### PR TITLE
Adds monitoring playbooks and dynamic inventory

### PR DESCRIPTION
**Pull request by denibertovic**
Also adds a basic makefile for calling the playbooks.
This should be replaced by a proper CLI down the road.

@mrgnr This should help streamline the customer installments for the time being. We might have to rename some folders in the various repos so that they're consistent. The makefile is only temporary until we build a proper cli tool.

### CODE REVIEW

**Comment by denibertovic** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;brQ5x5Ou&#x2F;335-6-write-small-playbook-for-edx-monitoring-roles"> (6) Write small playbook for edx monitoring roles</a>

File: playbooks/inventory [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        with open("../../edx-terraform/{platform}/{tier}/{customer}/state.tfstate".format(
+            platform=self.platform,
+            tier=self.tier,
+            customer=self.customer)) as f:
+
+            state_file = json.loads(f.read())
+            outputs = state_file["modules"][0]['outputs']
+            for k,v in outputs.items():
+                # name format from state file: "server_group_ips"
+                name = k.split("ips")[0][:-1]
```
Comment (mrgnr) [inline playbooks/inventory]: Any reason not to say `name = k.split("_ips")[0]`?

File: playbooks/Makefile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+.DEFAULT_GOAL = help
+
+## Install monitoring on customer cluster. Example: CUSTOMER=qa ENVIRONMENT=production TIER=basic PLATFORM=azure make monitoring
+monitoring:
+	@@if test -z "$$CUSTOMER"; then echo "ERROR: CUSTOMER is not defined."; exit 1; fi;
+	@@if test -z "$$PLATFORM"; then echo "ERROR: PLATFORM is not defined."; exit 1; fi;
+	@@if test -z "$$TIER"; then echo "ERROR: TIER is not defined."; exit 1; fi;
+	@@if test -z "$$ENVIRONMENT"; then echo "ERROR: ENVIRONMENT is not defined."; exit 1; fi;
+	@ansible-playbook -i inventory monitoring.yml --extra-vars "@../../edx-configs/$$CUSTOMER/$$ENVIRONMENT/server-vars.yml"
```
Comment (mrgnr) [inline playbooks/Makefile]: Caveat: Some customers in edx-configs are structured as `$CUSTOMER/$ENVIRONMENT/files/server-vars.yml`. We'll have to address this when refactoring edx-configs.

**Comment by mrgnr** [comment]
@denibertovic LGTM

File: playbooks/inventory (hunk 1 above)
Comment (denibertovic) [inline playbooks/inventory]: Good question. Yes! The reason is if we have names like "group_name_ips" then we would get "group_name_" and we want to remote the trailing "_" .

File: playbooks/Makefile (hunk 2 above)
Comment (denibertovic) [inline playbooks/Makefile]: Good catch! Yes we have to standardize the naming of this.

**Pull request merged**